src/analysis/utils/gce-machines.ts

```diff
@@ -153,6 +153,8 @@ export const zonesToGpus = [
   { name: 'US-WEST4-C', validTypes: [] },
 ];
 
+// **Deprecated**
+// Use cloudServiceTypes in `runtime-config-models`
```

Comment:
Not without changing the logic... the TODO on line 135 of modal-utils is necessary before. Currently, we use this instead of the `runtimeTypes` in a lot of conditional logic. Hopefully if there is time for a second phase of this it can be deleted, but more testing is needed first.